administrate/zone-migration.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Zone Migration
+tags:
+- migration
+- zone
+- application
+- addon
+---
+
+# Migrate your services from one zone to another
+
+In this document, we will see how you can migrate your Clever Cloud services from one zone to another zone, for example from Paris to Montreal.
+
+For most of the services, you will be able to migrate them as-is. For others, you will need to create them and do a manual migration.
+
+Keep in mind that our support team remains available should you have any questions regarding zone migration.
+
+## Application migration
+
+Most of the applications will be able to be migrated without issues. If your application matches any criteria is in the list below, you will need extra steps
+described in the following sub-sections.
+
+- Any application that uses a FSBucket add-on
+- PHP applications
+
+If your application matches any criterias on this list, please read the prerequisites below first and then follow the right sub-section.
+
+### Prerequisites
+
+Besides redeploying your application in your target zone, you have to keep a few things in mind:
+- Increased latency might be observed during DNS propagation (more below)
+- You will need to update your DNS settings so make sure you have access to your DNS registrar
+
+One of the first things you can do is to lower your DNS records TTL (time to live) to speed up the DNS propagation when you will update your domains DNS records.
+
+On your registrar's interface, find your application's domain. Usually, the DNS record will be `domain.tld IN CNAME domain.<current-zone>.clever-cloud.com` with a certain TTL.
+Update that TTL to 60 seconds instead and wait for this change to propagate (wait as much time as the old TTL value). This change will allow a faster DNS propagation when
```

Comment:
```suggestion
Edit that TTL to 60 seconds instead of the existing value, and wait for this change to propagate (wait as much time as the old TTL value). This change will allow a faster DNS propagation when
```